src/AccessToken.php

```diff
@@ -226,7 +231,7 @@ private function verifyEs256($token, array $certs, $audience = null)
      *                              the JWT.
      * @return array|bool the token payload, if successful, or false if not.
      */
-    private function verifyRs256($token, array $certs, $audience = null)
+    private function verifyRs256($token, array $certs, $audience = null, $issuer = null)
```

Comment:
It would be nice to add this to the docblock, otherwise LGTM! :)

Reply:
Done